src/Resources/contao/library/Contao/DcaLoader.php

```diff
@@ -85,7 +85,10 @@ public function load($blnNoCache=false)
 					include $file;
 				}
 			}
-			catch (\InvalidArgumentException $e) {}
+			catch (\InvalidArgumentException $e)
+			{
+				\System::getContainer()->get('monolog.logger.contao')->notice($e->getMessage(), ['exception' => $e]);
```

Comment:
But this will also log exceptions for every bundle or module that does not have a `dca/` folder! What exactly do you want to log?